CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ set(RGL_BUILD_TESTS ON CACHE BOOL
 set(RGL_BUILD_TOOLS ON CACHE BOOL "Enables building RGL executable tools")
 
 # Extensions configuration
+set(RGL_BUILD_ALL OFF CACHE BOOL
+        "Enables all possible extensions, tools and tests.")
```

Comment:
I'd prefer not to pollute CMakeLists with the flag `all`. We can manage it in the `setup.py` script.

Reply:
I am aware of ros2standalone build.
This should be discussed. IMHO in "build all" mode we should choose "default" ros2 building method

Reply:
@prybicki, what are your thoughts?